Anlab.Core/Models/OrderDetails.cs

```diff
@@ -23,7 +23,7 @@ public class OrderDetails
         [DataType(DataType.MultilineText)]
         public string LabComments { get; set; }
         public decimal AdjustmentAmount { get; set; }
-        public decimal GrandTotal { get
+        public decimal FinalTotal { get

```

Comment:
@laholstege I'm using the GrandTotal in my Payment PR. If it isn't too much of a problem, please change FinalTotal back to GrandTotal.